net_http_mux.go

```diff
@@ -69,7 +69,9 @@ func Patch[T, B any](s *Server, path string, controller func(ContextWithBody[B])
 	return registerFuegoController(s, http.MethodPatch, path, controller, options...)
 }
 
-// Register registers a controller into the default mux and documents it in the OpenAPI spec.
+// Register registers a controller into the default net/http mux.
+//
+// Deprecated: Used internally. Please satisfy the [Registerer] interface instead and pass to [Registers].
 func Register[T, B any](s *Server, route Route[T, B], controller http.Handler, options ...func(*BaseRoute)) *Route[T, B] {
```

Comment:
I assume this will either be not exported or moved to `netHttpRouteRegisterer.Registerer`